install.sh

```
@@ -0,0 +1,253 @@
+    esac
+}
+
+rc_has_marker() {
+    [ -f "$1" ] && grep -qF "$MARKER_BEGIN" "$1"
+}
+
+append_path_block() {
+    rc="$1"
+    {
+        printf '\n%s\n' "$MARKER_BEGIN"
+        printf 'export PATH="%s:$PATH"\n' "$DATUS_BIN_DIR"
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)